deck-service/src/main/java/magicthebuilder/deckservice/entity/User.java

```diff
@@ -0,0 +1,23 @@
+package magicthebuilder.deckservice.entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue
+    public Long id;
```

Comment:
I think that User ids shouldn't be generated in DeckService.

Reply:
True, it will be imported from user service, made it like that for testing purposes and forgot to change it.